app/views/home/error.html.erb

```diff
@@ -4,20 +4,20 @@
 
 <% if flash[:error] %><p><b><%= flash[:error] %></b></p><% end %>
 
-<p><b>Oops!</b> Autolab seems to have done something wrong.  We have been notified about this and will look into it. If you would like to file a Bug Report, you may email us at <a href="mailto:<%= Rails.configuration.school['support_email'] %>"><%= Rails.configuration.school['support_email'] %></a> or open an issue in our <a href="https://github.com/autolab/Autolab/issues/">issue tracker</a> on GitHub.</p>
+<p><b>Oops!</b> Autolab seems to have done something wrong. We have been notified about this and will look into it. If you would like to file a Bug Report, you may email us at <a href="mailto:<%= Rails.configuration.school['support_email'] %>"><%= Rails.configuration.school['support_email'] %></a> or open an issue in our <a href="https://github.com/autolab/Autolab/issues/">issue tracker</a> on GitHub.</p>
 
 <%= image_tag "DonkeyKong.jpg", width: '200px' %>
 
 <% if @error then %>
   <br><br>
-  <p><i>Remember that if you modify any config files, you'll need to  reload them for the changes to take effect.</i><br>
+  <p><i>Remember that if you modify any config files, you'll need to reload them for the changes to take effect.</i><br>
     <% if @course_name then %>
       <b><%= link_to "Reload Course Config",
-              reload_course_path(name: @course_name) %></b><br>
+              reload_course_path(name: @course_name), data: {method: "post"} %></b><br>
```

Comment:
I believe style checking exists for ruby files (`.rb`) but not for erb files (`.html.erb`). The best way to go about it would probably be to use a linter such as [erb-lint](https://github.com/Shopify/erb-lint).

Definitely something worth looking into, perhaps in another PR.

Reply:
Yeah something like erb-lint would be a good idea, but yes it would be good for another PR / issue.